app/src/main/java/com/babylon/wallet/android/data/gateway/generated/models/TransactionPreviewResponse.kt

```diff
@@ -45,7 +52,14 @@ data class TransactionPreviewResponse (
 //    val resourceChanges: kotlin.collections.List<@Contextual kotlin.Any>,
 
     @SerialName(value = "logs")
-    val logs: kotlin.collections.List<TransactionPreviewResponseLogsInner>
+    val logs: kotlin.collections.List<TransactionPreviewResponseLogsInner>,
 
-)
+    /* An optional field which is only provided if the `request_radix_engine_toolkit_receipt` flag is set to true when
+    requesting a transaction preview from the API. This receipt is primarily intended for use with the toolkit and
+    may contain information that is already available in the receipt provided in the `receipt` field of this response.
+    A typical client of this API is not expected to use this receipt. The primary clients this receipt is intended for
+    is the Radix wallet or any client that needs to perform execution summaries on their transactions.  */
+    @Contextual @SerialName(value = "radix_engine_toolkit_receipt")
+    val radixEngineToolkitReceipt: JsonElement? = null
```

Comment:
`radixEngineToolkitReceipt` is an opaque type. Kotlin doesn’t have explicit opaque types like those found in languages such as Swift or Rust. However, we can achieve similar functionality in Kotlin through the use of interfaces, abstract classes like we have done with other GW types.

However here we need just the string representation of this type thus defined it as `JsonElement`.